nixos/modules/programs/phosh-ui.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkOption mkIf types;
+  cfg = config.programs.phosh-ui;
+
+  oskItem = pkgs.makeDesktopItem {
```

Comment:
Could you add a comment linking to the corresponding file in librem base repo or whatever this is modelled upon.

Reply:
done